contrib/go/src/python/pants/contrib/go/tasks/go_test.py

```diff
@@ -105,6 +115,8 @@ def run_tests(self, fail_fast, test_targets, args_by_target):
       )
       go_cmd = self.go_dist.create_go_cmd('test', gopath=gopath, args=cmdline_args)
 
+      self.context.log.debug('go_cmd: {}'.format(go_cmd))
```

Comment:
This should be logged via the workunit I think? Not a problem to leave it though.

Reply:
Does `self.context.log.debug()` not do that already? Looking into `plaintext_reporter.py` I'm seeing that `_COLOR_BY_LEVEL` is `cyan` for debug logs, which is what this comes out as, and `self.context.log` ends up being `run_tracker.logger` (according to `context.py`), and in `run_tracker.py` the `def log(self)` method logs against `self._threadlocal.current_workunit`, which seems like it might do the right thing?

Reply:
It doesn't get rendered to stdout, but it does get rendered in `./pants server --open`, IIRC.

Reply:
I definitely see it in the output when I run pants with `-ldebug` -- maybe it goes to stderr?